### ISSUE TITLE
The Save a copy issue

### ISSUE THREAD

**Issue by wocmultimedia** [Sigil-Ebook/Sigil#477]
Hi, 
Maybe did it happen only to me but I made en ebook I saved it the first time and then I used the Save a copy feature. Well, after doing a complete revision work and corrections I closed the program. When I restart it again and opened the book it was at the very first version without any new correction.
I'm not sure if it is a bug of the Save a copy or if it is something related to Undo because latter doesn't show a list of previous actions. I clicked to revert one action and maybe when there's no action to revert it revert at the very beginning.
I point out at these two features because I restarted my revision again but this time using the save as and it works correctly.
I am using the 0.97 version
Thanks
Angelo

**Comment by dougmassay** [comment]
I've never had any trouble with the Save As function not working correctly.  Having said that, however, the 0.9.7 version of Sigil is quite old. Even if there was a bug in it, we wouldn't be able to address it unless it was also in the latest version.

If you ever find a way to consistently recreate this issue (in the latest version of Sigil), feel free to reopen this issue and provide the steps necessary to reproduce it.

**Comment by wocmultimedia** [comment]
I got the issue with the Save a copy feature not Save as. But maybe it happen to me only.

**Comment by dougmassay** [comment]
Well yes. Save a Copy does not work like  Save As. You would actually have to open the newly saved copy for editing after saving it Otherwise you're still editing the original. Not like Save As which actually switches you to the new epub where you can continue to edit. More than like, you merely got confused about which copy you were actually editing at the time.

**Comment by wocmultimedia** [comment]
Well, thanks a lot. I know the difference between the two features  but as you said maybe I made a mistake. It's also possible how I told before that the clicking once more the Undo button when there was no more fresh action to undo caused the very strange issue. I never experienced with Sigil. 
I use it since 2011 that's why I supposed could be a bug of this 0.97 version not to bother you.
I also found out that if you perform a find a replace action cannot revert in case of a mistake. Is it normal this, it is?
Thanks for the great job you are doing
